vertx-web/src/main/java/io/vertx/ext/web/impl/HandlersCollection.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Red Hat, Inc.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.web.impl;
+
+import io.vertx.core.Handler;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class HandlersCollection<E> {
```

Comment:
Just 3 lines of comment about the 3 key behaviours:
- it's not thread-safe
- it can keep on growing to accomodate new handlers and removal doesn't compact
- ID's/indexes are stable regardless add/remove

Reply:
Done